### PR TITLE
Extension presentation on android correction

### LINKED ISSUE #1736: Minor text update
Under the section heading "Effect on your add-on UI", on [this page](https://extensionworkshop.com/documentation/develop/differences-between-desktop-and-android-extensions/), there is a sentence that requires updating.

The sentence in question is:

"You cannot reveal your add-on through a sidebar or context menu. However, you can expose your extension as an option under the Add-ons item in the browser menu **or as an address bar button**."

This part of the sentence is no longer accurate and needs to be removed, "...**or as an address bar button**"

I vetted this with the Android Engineering team (Rob W)

### PR DESCRIPTION
**Pull request by rebloor**
Fixes #1736

### CODE REVIEW

**Comment by rebloor** [comment]
FYI @esullivan-moz

File: src/content/documentation/develop/differences-between-desktop-and-android-extensions.md [hunk 1]
```diff
@@ -94,22 +94,16 @@ Developer tools for Firefox for Android are provided through remote debugging me
...
 - for Manifest V3 with the `manifest.json` [`action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action) key and [`action`](https://developer.mozilla.org//Add-ons/WebExtensions/API/action) API.
 
-You can also use an address bar button (through the `manifest.json` [`page_action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action) key and [`pageAction`](https://developer.mozilla.org//Add-ons/WebExtensions/API/pageAction) API), remembering that this button is hidden by default and must be shown programmatically.
+Note that the [`browserAction`](https://developer.mozilla.org//Add-ons/WebExtensions/API/browserAction) and [`action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/action) popup content opens as an overlay, covering the browser window until the user closes the overlay.
 
-The features of [`pageAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction) in Firefox for Android differ slightly from the desktop version.
```
Comment (Rob--W) [inline src/content/documentation/develop/differences-between-desktop-and-android-extensions.md:106]: This part can be kept, as there is still a difference: on desktop, the pageAction button may appear inside the address bar, but on mobile, the button appears in the same place as the browserAction (MV2)/action (MV3) button.

**Review by Rob--W — COMMENTED** [review]
Could you rebase your branch to drop the extraneous commits?

**Comment by rebloor** [comment]
@Rob--W my attempts to re-base only seem to have made things worse, so I've created a [new PR](https://github.com/mozilla/extension-workshop/pull/1766)

**Pull request closed without merging**